packages/css-library/README.md

```diff
@@ -1,3 +1,17 @@
 # css-library
 
-This package is the early discovery work on basing VADS styles off of USWDS v3.
+## Stylesheets
+
+The stylesheets generated by this library are based on USWDS version 3 and are intended to eventually replace the Formation CSS library:
+
+- fonts.css
+- elements.css
+- utilties.css
+
+To generate stylesheets, run `yarn build:stylesheets` or `yarn build:minify`
+
+## Design Tokens
+
+The tokens generated by this library will be available in several different formats so that various tools and environments can use them.
+
+To generate CSS token files, run `yarn build:tokens`
```

Comment:
We will not want this to be a manual process forever but for now, I thought it would be useful to mention how to generate the files while we are still working on building the library.

We have a backlog ticket for figuring out how we want to [publish multiple packages here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1954).